pyproject.toml

```diff
@@ -80,6 +80,7 @@ dependencies = [
 
   # Preview
   "canals==0.8.0",
+  "openai",
```

Comment:
Remember to add this in [deepset-ai/haystack-preview-package](https://github.com/deepset-ai/haystack-preview-package).

Reply:
Already done, as noted in the PR description